packages/mermaid/src/docs.mts

```diff
@@ -41,6 +41,10 @@ import { remark } from 'remark';
 // @ts-ignore No typescript declaration file
 import flatmap from 'unist-util-flatmap';
 
+const version = (
```

Comment:
Very minor, but it's potentially worth renaming this variable to `mermaidMajorVersion` or `MERMAID_MAJOR_VERSION`.